### PR TITLE
[MAINTENANCE] Adding a test workflow and updating the `Makefile`

### PR DESCRIPTION
**Pull request by Mxrcon**
hey :wave:, as a small maintenance I tried to update the `TEST` workflow and updated the `Makefile` so we can run `make run_test` and test the complete run with `docker` and `conda`. 

This might be an initial solution, but after the v1.0.0 release we can discuss an implementation of pytests and md5 checking. What do you think about it? I tested it here and apparently, it's working well.

Feel free to add your thoughts on the comments, I'll available to make any requested changes, we can also discuss the makefile implementation as this still very initial.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,4 +7,5 @@ run_dev:
 	nextflow run main.nf -profile dev -resume -with-tower
 
 run_test:
-	nextflow run main.nf -params-file params/standard.yml -entry TEST -resume
+	nextflow run main.nf -params-file params/test.yml -entry TEST -resume -profile conda
```
Comment (abhi18av) [inline Makefile:10]:
```suggestion
	nextflow run main.nf -params-file params/test.yml -entry TEST -resume -profile standard,docker
```

File: Makefile [hunk 2]
```diff
@@ -7,4 +7,5 @@ run_dev:
...
+	nextflow run main.nf -params-file params/test.yml -entry TEST -resume -profile docker
```
[the remark this answers is not in the document]
Comment (abhi18av) [reply Makefile:11]: Chossing to stick only with `docker`

File: params/test.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+genomeIds:
+  - "ERR037481"
+  - "ERR216945"
+  - "ERR1193883"
+
+outdir: "./test_output"
+
+ncbi_api_key: "b6d95ddb1bbd179bf9408c2726dba5ddfe09"
```
Comment (abhi18av) [inline params/test.yml:8]: 🤦

**Review by abhi18av — APPROVED** [review]
Thanks for the PR Davi 🎉  🙏 

I'm merging this with some customizations.

**Comment by abhi18av** [comment]
> This might be an initial solution, but after the v1.0.0 release we can discuss an implementation of pytests and md5 checking. What do you think about it? I tested it here and apparently, it's working well.

Davi, for the time being, I think this would be overengineering on this pipeline since basically 3 tools are used. But definitely, I think that good mastery of testing nextflow pipeline is on our general roadmap (but not on this specific pipeline perhaps 🤔 )

File: params/test.yml (hunk 3 above)
Comment (Mxrcon) [reply params/test.yml]: I always forgot to remove my api keys before commiting :sob: I'll remove it from ncbi

**Comment by Mxrcon** [comment]
> I'd like to try out this https://github.com/askimed/nf-test

Yes! I'm really interested into learn testing and etc, maybe we can add this to WGS1 ou create a small repo like mtb-bioinformatics/modules and them try it!

**Comment by abhi18av** [comment]
I think that we're gonna take that approach, best to collaborate with nf-core/modules for the pipeline.

But that's long term, let's track this in a discussion perhaps?

**Comment by Mxrcon** [comment]
sure! completely agreed.

**Pull request merged**